src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
-          "$ref": "#/definitions/tool"
-        },
-
-        "run": 
-        {
-          "description": "A run object describes the invocation of the static analysis tool that produced the results specified in the containing runLog object.",
-          "$ref": "#/definitions/run"
-        },
-
-        "rules": 
-        {
-          "description": "An array of rule descriptor objects that describe all rules associated with an analysis tool or a specific run of an analysis tool.",
-          "type": "array",
-          "minItems": 1,
-          "items":
-          {
-            "$ref": "#/definitions/ruleDescriptor"
-          }
-        },
-
-        "results": 
-        {
-          "description": "The set of results contained in an SARIF log. The results array can be omitted when a runLog is merely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
-          "type": "array",
-          "minItems": 0,
-          "items":
-          {
-            "$ref": "#/definitions/result"
+            "default": [ ]
           }
         }
       },
-      "required": ["tool"]
+      "required": [ "tool" ]
     },
     "tool": 
     {
-      "description": "Information about the tool or tool pipeline that generated the results in this log.",
+      "description": "Describes that static analysis tool that was run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
         "name": 
         {
-          "description": "The name of the tool or tool pipeline that generated the results in this log, e.g., FxCop.",
+          "description": "The name of the tool.",
           "type": "string"
         },
 
         "fullName": 
         {
-          "description": "The name of the tool along with its version and any other useful identifying information, such as its locale, e.g., 'CodeScanner 2.0, Developer Preview (en-US)'.",
+          "description": "The name of the tool along with its version and any other useful identifying information, such as its locale.",
           "type": "string"
         },
 
         "version": 
         {
-          "description": "A version that refers to the tool as a whole (as opposed to, for example, the build version of an individual binary in the tool).",
+          "description": "The tool version.",
           "type": "string"
         },
 
         "fileVersion": 
         {
-          "description": "For operating systems (such as Windows) that provide the data, the binary version of the primary tool exe.",
+          "description": "For operating systems (such as Windows) that provide such data, the binary version of the tool's primary executable file.",
```

Comment:
"The binary version of the tool's primary executable file (for operating systems such as Windows that provide that information)."

---
In reply to: [59471363](https://github.com/Microsoft/sarif-sdk/pull/178#discussion_r59471363) [](ancestors = 59471363)